static/src/js/components/KeyboardShortcutsModal/KeyboardShortcutsModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Modal, Container,
+  Row, Col, Button
+} from 'react-bootstrap'
+import config from './config.json'
```

Comment:
Remove in favor of inline content. You can use a `const` to define the variable to match the currently implementation or if you'd like, you can move the content directly into the `render` function.